### PR TITLE
QOL: Move skin debug setting from each controller skin to a global setting

### PR DESCRIPTION
**Pull request by ianclawson**
This is mainly for ease of use in testing out new Controller Skins, as well as efforts in updating the skin spec.

DeltaCore PR: https://github.com/rileytestut/DeltaCore/pull/34

Trello Card: https://trello.com/c/IGSv3tcf

### CODE REVIEW

File: Delta/Emulation/GameViewController.swift [hunk 1]
```diff
@@ -281,6 +281,7 @@ extension GameViewController
         self.view.layoutIfNeeded()
         
         self.controllerView.translucentControllerSkinOpacity = Settings.translucentControllerSkinOpacity
+        self.controllerView.isControllerSkinDebugModeEnabled = Settings.isControllerSkinDebugModeEnabled
```
Comment (rileytestut) [inline Delta/Emulation/GameViewController.swift:284]: As mentioned in DeltaCore PR, I think this should be `.isDebugModeEnabled`
```suggestion
        self.controllerView.isDebugModeEnabled = Settings.isControllerSkinDebugModeEnabled
```

File: Delta/Settings/SettingsViewController.swift [hunk 2]
```diff
@@ -99,6 +99,12 @@ class SettingsViewController: UITableViewController
             self.versionLabel.text = NSLocalizedString("Delta", comment: "")
             #endif
         }
+        
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(SettingsViewController.handleTapGesture(_:)))
+        tapGesture.numberOfTapsRequired = 13
```
Comment (rileytestut) [inline Delta/Settings/SettingsViewController.swift:104]: Happy to have this feature hidden for now, but 13 might be a bit high even for a hidden feature. IMO ~3 taps would be enough for a relatively harmless feature like this.
```suggestion
        tapGesture.numberOfTapsRequired = 3
```

**Review by rileytestut — CHANGES_REQUESTED** [review]

**Comment by ianclawson** [comment]
@rileytestut thanks for the review! Agree with your suggestions and just implemented them, ready for another pass 👍

**Comment by ianclawson** [comment]
Closing this to clean up my work list of Open PRs

**Comment by ianclawson** [comment]
Closing this to clean up my work list of Open PRs

**Pull request closed without merging**